### PR TITLE
Platfrom 310

### PR DESCRIPTION
**Pull request by jamccomb92**
### Description of proposed changes
_What does this pull request accomplish? Why should it be integrated?_
CCreates script for updating latest version number with npm run post-version
### Checklist
_Put an `x` in the boxes that apply._
- [x] My lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my changes work (if applicable)
- [ ] I have added or updated necessary documentation (if appropriate)

### Additional comments
_Is there anything else you want us to know?_

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -64,6 +65,7 @@
     "lodash": "4.17.21",
     "native-base": "2.13.8",
     "parse": "2.14.0",
+    "prompt": "1.1.0",
```
Comment (hopetambala) [inline package.json:68]: @jamccomb92 make sure this is in `devDependencies`. Libraries in `dependencies` get bundled and built with the app we deploy to the stores. Libraries in `devDependencies` are libraries for development like linting, testing, releasing etc

**Review by hopetambala — CHANGES_REQUESTED** [review]
move that library and i'll check locally! Look great so far though

File: package.json (hunk 1 above)
Comment (jamccomb92) [reply package.json]: @hopetambala moved to devDependencies, but got a linting error saying it should be in dependencies.

Comment (hopetambala) [reply package.json]: oh weird okay i'll check this out

**Review by hopetambala — APPROVED** [review]

**Review by hopetambala — APPROVED** [review]

**Pull request merged**